spec/spec_helper.rb

```diff
@@ -38,6 +38,7 @@
     # set to strictest setting for testing
     # by default Puppet runs at warning level
     Puppet.settings[:strict] = :warning
+    Puppet.settings[:strict_variables] = true
```

Comment:
Isn't this default? https://github.com/puppetlabs/puppetlabs_spec_helper/blob/121a67515480c15561bcc67271dab4a82f87ffa8/lib/puppetlabs_spec_helper/module_spec_helper.rb#L71